### PR TITLE
Adds Score to GameScreen

### PR DESCRIPTION
**Pull request by hareshkh**
The objective of this pull request is to create scoring criteria for the game and render the same onto the Game Screen.

### CODE REVIEW

File: core/src/com/sdsmdg/kd/gameworld/GameRenderer.java [hunk 1]
```diff
@@ -21,6 +26,9 @@ public GameRenderer (GameWorld gameWorld) {
         cam.setToOrtho(true, Main.screen.x, Main.screen.y);
         shapeRenderer = new ShapeRenderer();
         shapeRenderer.setProjectionMatrix(cam.combined);
+        bitmapFont = new BitmapFont();
+        batch = new SpriteBatch();
+        margin = (15*(Main.screenArea)/(1920*1080));
```
Comment (karandesai-96) [inline core/src/com/sdsmdg/kd/gameworld/GameRenderer.java]: 1920*1080 ? why such numbers ? And please be generous with spaces, `(15 * Main.screenArea) / (1920 * 1080));`

File: core/src/com/sdsmdg/kd/gameworld/GameRenderer.java [hunk 2]
```diff
@@ -66,5 +74,10 @@ public void render() {
         shapeRenderer.setColor(new Color(0xf44336ff));
         shapeRenderer.circle(gameWorld.magnus.x, gameWorld.magnus.y, gameWorld.magnus.radius);
         shapeRenderer.end();
+
+        batch.begin();
+        bitmapFont.setColor(0.1f, 0.1f, 0.1f, 1.0f);
+        bitmapFont.draw(batch, "Score :"+gameWorld.gameScoreToDisplay, margin, Main.screen.y-margin);
+        batch.end();
```
Comment (karandesai-96) [inline core/src/com/sdsmdg/kd/gameworld/GameRenderer.java]: I don't remember exactly, but I guess batch was supposed to be disposed or something, can you confirm ?

File: core/src/com/sdsmdg/kd/gameworld/GameWorld.java [hunk 3]
```diff
@@ -112,6 +123,21 @@ else if (currentWeapon == 5) {
...
+
+            if (gameScore%1000 == 0){
+                multiplier++;
+                step = 0.1f;
+            }
+            else{
+                if (gameScore%100 == 0){
+                    step += 0.05f;
+                }
+            }
```
Comment (karandesai-96) [inline core/src/com/sdsmdg/kd/gameworld/GameWorld.java]: This logic looks good. :smile:

File: core/src/com/sdsmdg/kd/gameworld/GameRenderer.java (hunk 1 above)
Comment (hareshkh) [inline core/src/com/sdsmdg/kd/gameworld/GameRenderer.java]: 1920*1080 is the screen resolution I tested it on.

**Pull request merged**